frontend/src/lib/k8s/apiProxy.ts

```diff
@@ -55,6 +115,9 @@ export async function request(
   let fullPath = path;
   if (useCluster && cluster) {
     const token = getToken(cluster);
+
+    await refreshToken(token);
```

Comment:
Could this possibly lead to a token being refreshed in less than a fraction of seconds if we have "simultaneous" requests throughout Headlamp? Maybe we'll have to guard for that, otherwise we may get a token refreshed multiple times in a very short time span.